README.md

```diff
@@ -1,37 +1,62 @@
 # Exercism Elm Track
-[![build status](https://github.com/exercism/elm/workflows/elm%20%2F%20master/badge.svg)]
```

Comment:
There were square brackets appearing round the image / badge on the github ui, so I removed them